packages/block-editor/src/components/block-toolbar/index.js

```diff
@@ -120,6 +126,7 @@ export function PrivateBlockToolbar( {
 				_isDefaultEditingMode,
 			isUsingBindings: _isUsingBindings,
 			canRemove: canRemoveBlock( selectedBlockClientId ),
+			selectedHasTemplateLock: _hasTemplateLock,
```

Comment:
Nit: Naming thing (also applies to other files) - `selected` isn't needed as the toolbar only shows for selected blocks. `hasTemplateLock` implies any kid of template lock (that it could also be 'all'), but that isn't the case.

Maybe it can be simplified to `isContentOnly` or `hasContentOnlyLocking`